src/config/konnectors.json

```diff
@@ -1160,6 +1160,7 @@
     "editor": "Brice Coquereau",
     "vendorLink": "https://clients.direct-energie.com/connexion-clients-particuliers/",
     "categories": ["energy"],
+    "testing": true,
```

Comment:
Is it normal that you put Direct energie in `tesint` mode here?

Reply:
Not at all, it was for enercoop